proxy/certs/certs.go

```diff
@@ -310,7 +322,10 @@ func (s *RemoteCertSource) Remote(instance string) (cert *x509.Certificate, addr
 	req := s.serv.Connect.Get(p, regionName)
 
 	var data *sqladmin.ConnectSettings
-	err = backoffAPIRetry("get instance", instance, func() error {
+	err = backoffAPIRetry("get instance", instance, func(timestamp time.Time) error {
```

Comment:
nit: consider changing timestamp to "stale_read_time" (since that's what it actually is)

Reply:
Good idea.